xsimd.pc.in

```diff
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
-libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
-includedir=${prefix}/include
+ibdir=@libdir_for_pc_file@
```

Comment:
@serge-sans-paille I’m very sorry for procrastinating on this, but there seems to be a typo here:

```suggestion
libdir=@libdir_for_pc_file@
```

Reply:
> @serge-sans-paille I’m very sorry for procrastinating on this, but there seems to be a typo here:

I'm very sorry for the typo :-/ 
Fixed as of ed0ad83a0dff7fb38a6e034e56e1f12d6d01ac6a